tests/conftest.py

```diff
@@ -2691,8 +2691,11 @@ def ceph_toolbox(request):
     teardown = config.RUN['cli_params'].get('teardown')
     skip_ocs = config.ENV_DATA['skip_ocs_deployment']
     if not (deploy or teardown or skip_ocs):
-        # Creating toolbox pod
-        setup_ceph_toolbox()
+        try:
```

Comment:
I think the better approach will be to check if OCS is installed and if not, we will add it to the condition above and will not run this block instead of try expect here.

It can be handled in different PR if it's blocking uninstall tests. Please open issue if it's going to be merged.